components/DisplaySettings/index.js

```diff
@@ -0,0 +1,44 @@
+export default function DisplaySettings({ settings, changeSettings }) {
+  return (
+    <>
+      <h3>Display Settings</h3>
+      <label htmlFor="checkboxLow">Low</label>
+      <input
+        id="checkboxLow"
+        checked={settings.impactLow}
+        onChange={() => changeSettings("impactLow")}
+        type="checkbox"
+      />
+      <label htmlFor="checkboxMedium">Medium</label>
+      <input
+        type="checkbox"
+        id="checkboxMedium"
+        checked={settings.impactMedium}
+        onChange={() => changeSettings("impactMedium")}
+      />
+      <label htmlFor="checkboxHigh">High</label>
+      <input
+        type="checkbox"
+        id="checkboxHigh"
+        checked={settings.impactHigh}
+        onChange={() => changeSettings("impactHigh")}
+      />
```

Comment:
It is not clear what Low/Medium/High means when looking at the interface. Maybe "Show High Impact Events", etc.?